app/backend/aquifers/models.py

```diff
@@ -259,4 +246,4 @@ class Meta:
         verbose_name_plural = 'Aquifers'
 
     def __str__(self):
-        return '{} - {}'.format(self.code, self.description)
\ No newline at end of file
+        return '{} - {}'.format(self.code, self.description)
```

Comment:
Note that Ryan fixed this at: 
https://github.com/bcgov/gwells/commit/cc6eed0734dae8590861ed9b6e9876bb0e83f709

There will be a subsequent PR from @rfb  later.